docs/user-documentation/glossary.md

```diff
@@ -1,95 +1,202 @@
  
 The Fedora repository functions as the standard _smart storage_ for Islandora.
  
 ## Field
-Data of a certain type that is attached to a content entity. For instance, on a Resource Node content type, you might have fields for a title, description, display hints, subjects, and other metadata. 
+Data of a certain type that is attached to a content entity. For instance, on a Resource Node [content type](#content-type), you might have fields for a title, description, display hints, subjects, and other metadata. 
  
+## Field instance
+
+## Field type
+
+## FITS
+File Information Tool Set, a set of software components for identifying, validating and extracting of technical metadata for a wide range of file formats.
+
+## Fixity
+Also file fixity; digital preservation term meaning that a digital file remains unchanged ('fixed') over time. Fixity checking verifies that a file has not been corrupted or manipulated during a transfer process or while being stored. Typically, a fixity checking process computes [checksums](#checksum) or cryptographic hashes for a file and compares the result to a reference value stored earlier. The contributed Islandora module 'RipRap' supports fixity checking and error reporting.
+
 ## Flysystem
-Flysystem is a filesystem abstraction library for PHP. Islandora uses Flysystem to swap about different backend filesystem applications. Islandora provides a custom Flysystem adapter for Fedora.
+Flysystem is a filesystem abstraction library for PHP. Islandora uses Flysystem to swap about different backend filesystem applications. Islandora provides a custom Flysystem adapter for [Fedora](#fedora-repository-software).
+
+## GLAM
+Acronym for "galleries, libraries, archives, and museums".
 
 ## IIIF
 The [International Image Interoperability Framework](https://iiif.io/). A set of open standards that help archives, libraries, and museums make the most of their digitized collections with deep zoom, annotation capabilities, and more, and also the community of users and developers that suport the framework.
  
 ## Ingest
-To ingest an object is to add an entry for it in Islandora. This is done primarily through the Drupal interface. 
+To ingest an object is to add an entry for it in Islandora. This is done primarily through the [Drupal](#drupal) interface. 
  
 The word 'ingest' is used in repository language because repository software, rather than having an object and associated components forced upon it, takes in and processes the object itself, creating derivatives automatically. Often, when referring to the process of ingesting objects, Islandora and its associated documentation use other terms such as 'import' or 'add resource node'. In such contexts, these terms generally refer to the ingest process.
```

Comment:
That was language I hadn't touched, thanks for pointing out the somewhat strange metaphor. How does this sound:

> In the context of digital repositories, ingest refers to the process by which the repository software imports and subsequently processes an object, creating derivatives automatically, and running any other processing that is configured to occur when an object is added. This would be distinguished by software which simply stores objects after import (with or without associated files) and performs no processing. The Islandora GUI and the documentation sometimes use other terms such as 'import' or 'add resource node'. In such contexts, these terms generally refer to the ingest process.

Reply:
Huge improvement! Thanks!